### ISSUE TITLE
Unable to combine rules

### ISSUE THREAD

**Issue by HMilbradt** [getjerry/nest-casl#260]
ConditionsProxy.get currently combins all the conditions into a single object.  This prevents you from declaring rules like the following.

Ex: Anyone can access public uploads, and users can access their own uploads

```
everyone({ can }) {
  can(Actions.read, Upload, { public: true });
},

user({ can, cannot, user }) {
  can(Actions.read, Upload, { user: user.id, public: false });
},
```

Calling `conditions.get()` produces the following object:

```
{ user: '4663a9ea-627c-483e-8e28-88739ffa0dc0', public: true }
```

Is there a possible way to work around this?  

The problematic code can be found here:

https://github.com/getjerry/nest-casl/blob/1e785bb7b97d2cfaf1b3757393e5bd844f2b99e4/src/proxies/conditions.proxy.ts#L11

Thanks!

**Comment by liquidautumn** [comment]
Looks like a bug, I'll check asap.

**Comment by bouceka** [comment]
Any progress? I've been waiting for this fix.

**Comment by liquidautumn** [comment]
Fixed in [v1.6.13](https://www.npmjs.com/package/nest-casl/v/1.6.13)
